Obsidian/Client.cs

```diff
@@ -39,7 +39,7 @@ public sealed class Client : IDisposable
     private MinecraftStream minecraftStream;
     private readonly NetworkStream socketStream;
 
-    private Config config;
+    private ServerConfiguration config;
```

Comment:
again, any specific reason to rename? is it just for clarity? I'm down with merging this nonetheless but I just want to know the exact reasoning.

Reply:
See [previous comment.](https://github.com/ObsidianMC/Obsidian/pull/281#discussion_r973659601)